docs/source/Configuration.rst

```diff
@@ -111,6 +111,12 @@ configuration, after all the files are parsed in their correct resolution order.
 The command also shows the order in which your config files were parsed, giving you a better understanding of
 what's going on. The Section.Key nomenclature was inspired in ``git config --list`` output.
 
+Virtualenv
+==========
+
+When avocado runs inside virtualenv than path for global config files is also changed.
+For example, `avocado.conf` comes from the virual-env path `venv/etc/avocado/avocado.conf`.
+
```

Comment:
I don't think new section for virtalenv in configuration section is necessary. Anyway this note can be useful so how about adding second note in the `parsing order recap` just after the  `git` note mentioning the virtualenv?